baseline.gradle

```diff
@@ -78,7 +78,7 @@ subprojects {
     tasks.withType(JavaCompile).configureEach {
       options.errorprone.errorproneArgs.addAll (
           // error-prone is slow, don't run on tests/generated-src/generated
-          '-XepExcludedPaths:.*/(test|generated-src|generated)/.*',
+          '-XepExcludedPaths:.*/(test|integration|generated-src|generated)/.*',
```

Comment:
When I moved Azure tests into integration, I saw `IntLongMath:ERROR`. The check is disabled for test sources, so I assume it's safe to disable for integration test compilation too. Please let me know if you think otherwise.